File: src/main/java/com/huirong/hadoop/HelloWord.java
package com.huirong.hadoop;

import org.apache.hadoop.conf.*;
import org.apache.hadoop.fs.Path;



/**
 * Created by Huirong on 17/1/3.
 */
public class HelloWord {
    public static void main(String[] args) {
        Configuration conf = new Configuration();
        Path path = new Path("file:///Users/quixeynew/1.log");
    }

}

File: src/main/scala/com/huirong/HelloWord.scala
package com.huirong

import org.apache.spark.{SparkConf, SparkContext}

/**
  * Created by Huirong on 17/1/4.
  */
object HelloWord {
  def main(args: Array[String]): Unit = {
    val conf = new SparkConf().setAppName("shabi")
    val sc = new SparkContext(conf)
    val data = sc.textFile("file:///home/huirong/work/spark-1.6.1-bin-hadoop2.6/README.md")
    data.count()

    println("-----------")
    sc.stop()
  }

}
